CompressImagesFunction/CompressImagesFunction.cs

```diff
@@ -130,12 +130,14 @@ public static class CompressImagesFunction
             }
             else if (didCompress)
             {
+                var update = compressImagesMessage.IsRebase;
                 logger.LogInformation("CompressImagesFunction: Successfully compressed images for {Owner}/{RepoName}", compressImagesMessage.Owner, compressImagesMessage.RepoName);
                 openPrMessages.Add(new OpenPrMessage
                 {
                     InstallationId = compressImagesMessage.InstallationId,
                     RepoName = compressImagesMessage.RepoName,
                     CloneUrl = compressImagesMessage.CloneUrl,
+                    Update = update,
```

Comment:
Here we can assign directly
```suggestion
                    Update = compressImagesMessage.IsRebase,
```